.env.example

```
@@ -0,0 +1 @@
+API_URL=
```

```suggestion
API_URL=https://exmaple.com/
```

You should add an example so that others know if the `http://` protocol should be included, the `/` at then end, etc...